ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/logging.yml

```diff
@@ -1,12 +1,12 @@
 ### Logging ###
 logging:
-  console: debugging
+  console: error
   buffered:
     size: 1000000
-    level: informational
-  trap: informational
+    level: warnings
+  trap: disabled
   synchronous:
-    level: error
+    level:
```

Comment:
To test default behavior, we can simply do:

``` synchronous:``` instead of ```synchronous.level:```

Reply:
True! let me wait for the answer from Claus regarding the default value topic, will change it if I have to go back to the code anyway

Reply:
sure. You don't need to test this one anyway

Reply:
I think we should keep the default as is, so please test it.